src/lib/UI/Value/ValueFactory.php

```diff
@@ -248,10 +249,8 @@ public function createBookmark(Location $location): UIValue\Location\Bookmark
         return new UIValue\Location\Bookmark(
             $location,
             [
-                'contentType' => $this->contentTypeService->loadContentType($location->contentInfo->contentTypeId),
-                'pathLocations' => $this->pathService->loadPathLocations(
-                    $this->locationService->loadLocation($location->contentInfo->mainLocationId)
-                ),
+                'contentType' => $location->getContent()->getContentType(),
+                'pathLocations' => $this->pathService->loadPathLocations($location),
```

Comment:
NOTE: I assumed we don't need to load main location here but rather want path for the location bookmark is for.